app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -288,26 +293,24 @@ class PipelineSampleReport extends React.Component {
     );
   }
 
-  applySearchFilter(
-    searchTaxonId,
-    includedCategories,
-    input_taxons,
-    includedSubcategories = []
-  ) {
+  applySearchFilter(searchTaxonId = this.state.search_taxon_id) {
+    let input_taxons = this.state.thresholded_taxons;
+    let includedCategories = this.state.includedCategories;
+    let includedSubcategories = this.state.includedSubcategories;
+
     let selected_taxons = [];
-    const thresholded_taxons = input_taxons || this.state.thresholded_taxons;
-    const active_thresholds = this.state.activeThresholds;
     const specificOnly = this.state.readSpecificity === 1;
 
     if (searchTaxonId > 0) {
       let genus_taxon = {};
       let matched_taxons = [];
-      for (let i = 0; i < this.state.taxonomy_details.length; i++) {
-        const taxon = this.state.taxonomy_details[i];
+      let new_input_taxons = [];
```

Comment:
Just trying to understand the changes: what did taxonomy_details contained (vs thresholded_taxons) before?

Reply:
taxonomy_details contains ALL the taxons, while thresholded taxons only contains ones passing the threshold filters. so this is part of the change that text search won't return anything not already visible in the table based on the other filters